rules-reviewed/camel3/camel2/xml-renamed-components.windup.xml

```diff
@@ -156,7 +247,20 @@
                 </hint>
```

Comment:
Shouldn't it be the other way around the quickfix ?

Reply:
Done, thanks!